crates/iceberg/src/spec/manifest.rs

```diff
@@ -0,0 +1,1821 @@
+        assert_eq!(
+            entry.data_file.value_counts,
+            HashMap::from([
+                (4, 1),
+                (5, 1),
+                (2, 1),
+                (0, 1),
+                (3, 1),
+                (6, 1),
+                (8, 1),
+                (1, 1),
+                (10, 1),
+                (7, 1),
+                (9, 1)
+            ])
+        );
+        assert_eq!(
+            entry.data_file.null_value_counts,
+            HashMap::from([
+                (1, 0),
+                (6, 0),
+                (2, 0),
+                (8, 0),
+                (0, 0),
+                (3, 0),
+                (5, 0),
+                (9, 0),
+                (7, 0),
+                (4, 0),
+                (10, 0)
+            ])
+        );
+        assert!(entry.data_file.nan_value_counts.is_empty());
+        assert!(entry.data_file.lower_bounds.is_empty());
+        assert!(entry.data_file.upper_bounds.is_empty());
+        assert!(entry.data_file.key_metadata.is_empty());
+        assert_eq!(entry.data_file.split_offsets, vec![4]);
+        assert!(entry.data_file.equality_ids.is_empty());
+        assert_eq!(entry.data_file.sort_order_id, None);
+    }
+
+    #[test]
+    fn test_parse_manifest_v1_unpartition() {
+        let path = format!(
+            "{}/testdata/unpartition_manifest_v1.avro",
+            env!("CARGO_MANIFEST_DIR")
+        );
+        let bs = fs::read(path).expect("read_file must succeed");
+        let manifest = Manifest::parse_avro(bs.as_slice()).unwrap();
+        // test metadata
+        assert!(manifest.metadata.schema_id == 0);
+        assert_eq!(manifest.metadata.schema, {
+            let fields = vec![
+                Arc::new(NestedField::optional(
+                    1,
+                    "id",
+                    Type::Primitive(PrimitiveType::Int),
+                )),
+                Arc::new(NestedField::optional(
+                    2,
+                    "data",
+                    Type::Primitive(PrimitiveType::String),
+                )),
+                Arc::new(NestedField::optional(
+                    3,
+                    "comment",
+                    Type::Primitive(PrimitiveType::String),
+                )),
+            ];
+            Schema::builder()
+                .with_schema_id(1)
+                .with_fields(fields)
+                .build()
+                .unwrap()
+        });
+        assert!(manifest.metadata.partition_spec.fields.is_empty());
+        assert!(manifest.metadata.content == ManifestContentType::Data);
+        assert!(manifest.metadata.format_version == FormatVersion::V1);
+        assert_eq!(manifest.entries.len(), 4);
+        let entry = &manifest.entries[0];
+        assert!(entry.status == ManifestStatus::Added);
+        assert!(entry.snapshot_id == Some(2966623707104393227));
+        assert!(entry.sequence_number.is_none());
+        assert!(entry.file_sequence_number.is_none());
```

Comment:
![image](https://github.com/apache/iceberg-rust/assets/1134248/3c06da04-c8ed-4342-95d2-64e9fb17f213)

These should default to `0`. You could implement this using the initial-value of Iceberg:

![image](https://github.com/apache/iceberg-rust/assets/1134248/54fe8613-2b3d-4418-b6b8-6f593543efbc)


```suggestion
        assert!(entry.sequence_number == Some(0));
        assert!(entry.file_sequence_number == Some(0));
```